src/redis_db.cc

```diff
@@ -383,10 +386,8 @@ rocksdb::Status Database::FindKeyRangeWithPrefix(const std::string &prefix,
   // it's ok to increase the last char in prefix as the boundary of the prefix
   // while we limit the namespace last char shouldn't be larger than 128.
   std::string next_prefix = prefix;
-  char last_char = next_prefix.back();
-  last_char++;
-  next_prefix.pop_back();
-  next_prefix.push_back(last_char);
+  auto prefix_size = prefix.size();
+  next_prefix[prefix_size - 1] ++;
```

Comment:
```suggestion
  next_prefix[prefix_size - 1]++;
```

Reply:
It's my pleasure. Thx for the contributes of @ShooterIT .